lisa/node.py

```diff
@@ -161,6 +175,25 @@ def is_linux(self) -> bool:
         self.initialize()
         return self.os.is_linux
 
+    @property
+    def support_sudo(self) -> bool:
+        self.initialize()
+
+        # check if sudo supported
+        if self.is_linux and self._support_sudo is None:
+            process = self._execute("command -v sudo", shell=True, no_info_log=True)
+            result = process.wait_result(10)
```

Comment:
does it return immediately?

Reply:
yes, it's synchronous call, return immediately. If it spends more than 10 seconds, there must be some problem.